### PR TITLE
Newsletter 221 add french translation

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by bitschmidty** [comment]
@Copinmalin Can you correct the build errors:

> 2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:17: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:28: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:54: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:64: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:80: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:90: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:94: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:110: MD009 Trailing spaces
2:07:07 AM: _posts/fr/newsletters/2022-10-12-newsletter.md:117: MD009 Trailing spaces

**Comment by Copinmalin** [comment]
thanks @bitschmidty ,

I have a problem on my PC, when I try "make production", as result I have an error with ruby :-(

`bundle exec jekyll clean
Configuration file: /mnt/c/Users/cyril/OneDrive/Documents/GitHub/bitcoinops.github.io/_config.yml
           Cleaner: Nothing to do for /mnt/c/Users/cyril/OneDrive/Documents/GitHub/bitcoinops.github.io/_site.
           Cleaner: Nothing to do for /mnt/c/Users/cyril/OneDrive/Documents/GitHub/bitcoinops.github.io/.jekyll-metadata.
           Cleaner: Nothing to do for .sass-cache.
bundle exec _contrib/schema-validator.rb _data/schemas/compatibility.yaml _data/compatibility/blockchaininfo.yaml
/usr/bin/env: ‘ruby\r’: No such file or directory
make: *** [Makefile:120: _site/en/compatibility/blockchaininfo/index.html] Error 127`

**Comment by bitschmidty** [comment]
@Copinmalin Unfortunately, I have not run into that error myself before. But perhaps troubleshooting along these lines might help:

> https://stackoverflow.com/questions/29073826/env-ruby-r-no-such-file-or-directory

**Comment by Copinmalin** [comment]
@bitschmidty 
I've make change needed with trailing space,

I progress a little bit with error, but I have more problem now :,-(

**Comment by Copinmalin** [comment]
@bitschmidty : OK for me, thanks

**Comment by bitschmidty** [comment]
@Copinmalin squashed commits and merged. You are free to delete the branch, yes!

File: _posts/fr/newsletters/2022-10-12-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  jusqu'à plusieurs mois sans risquer de perdre des fonds avec leurs partenaires
+  de canaux. Bien que cela soit techniquement possible dans le protocole LN actuel,
+  cela dépendrait de la définition du paramètre 'settlement-delay' à des valeurs
+  élevées qui permettraient à un utilisateur malheureux ou à un accident d'empêcher
+  les fonds de plus d'une douzaine de canaux d'être utilisés pour ces mêmes mois.
+  Law propose de mitiger ce problème à travers deux modifications du protocole:
+
+    - *Triggered HTLCs:* Dans une norme [HTLC][topic htlc] utilisée pour
+      le paiement, Alice offre à Bob une certaine quantité de BTC s'il est
+      capable de publier une *préimage* inconnue auparavant pour un condensé
```
Comment (acidbunny21) [inline _posts/fr/newsletters/2022-10-12-newsletter.md:33]:
```suggestion
      capable de publier une *préimage* inconnue auparavant pour un condensat
```

File: _posts/fr/newsletters/2022-10-12-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        Bob de son intention de récupérer l'argent dans son portefeuille en
+        faisant confirmer une transaction *trigger* sur la chaîne. Ce n'est
+        que lorsque la transaction de déclenchement aura été confirmée par un
+        certain nombre de blocs (ou pendant une certaine durée) qu'Alice
+        pourra dépenser l'argent.
+
+        Bob serait assuré de pouvoir réclamer ses fonds à tout moment jusqu'à
+        ce que la transaction de déclenchement ait reçu le nombre convenu de
+        confirmations, même si des mois se sont écoulés depuis qu'un HTLC
+        normal aurait expiré. Si Bob est correctement indemnisé pour son attente,
```
Comment (acidbunny21) [inline _posts/fr/newsletters/2022-10-12-newsletter.md:49]:
```suggestion
        normal ait expiré. Si Bob est correctement indemnisé pour son attente,
```

File: _posts/fr/newsletters/2022-10-12-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  q3="Les nœuds DNS d'amorçage sont censés être réactifs et servir les adresses actualisées
+des nœuds Bitcoin. Pourquoi cela n'aide-t-il pas un noeud `-onlynet=tor?`"
+  a3="Les nœuds DNS d'amorçage ne fournissent que des adresses IPv4 et IPv6;
+ils ne sont pas en mesure de fournir un autre type d'adresse."
+  a3link="https://bitcoincore.reviews/26114#l-35"
+%}
+
+## Mises à jour release candidate
+
+*Nouvelles mises à jour et release candidates du principal logiciel d'infrastructure Bitcoin.
```
Comment (acidbunny21) [inline _posts/fr/newsletters/2022-10-12-newsletter.md:163]:
```suggestion
*Nouvelles mises à jour et release candidates des principaux logiciels d'infrastructure Bitcoin.
```

File: _posts/fr/newsletters/2022-10-12-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+Prévoyez s'il vous plait de vous mettre à jour à la nouvelle version ou d'aider à tester
```
Comment (acidbunny21) [inline _posts/fr/newsletters/2022-10-12-newsletter.md:164]:
```suggestion
Prévoyez s'il vous plait de vous mettre à jour aux nouvelles versions ou d'aider à tester
```

**Review by acidbunny21 — APPROVED** [review]

**Pull request merged**